pages/1.12/cli/index.md

```diff
@@ -8,137 +8,63 @@ excerpt: Understanding the command line interface utility in DC/OS
+After following the login flow, your CLI is now ready to interact with your cluster. You will notice that it now has additional commands such as `marathon`, `node`, `package` etc. These commands come from the [plugins](/1.12/cli/plugin), dcos-core-cli and, if applicable, dcos-enterprise-cli, which get automatically installed as part of the setup command.
 
-- After conversion to the [new configuration structure](#configuration-files), `DCOS_CONFIG` is no longer honored.
-- Before you call `dcos cluster setup`, you can change the configuration pointed to by `DCOS_CONFIG` using `dcos config set`. This command prints a warning message saying the command is deprecated and recommends using `dcos cluster setup`.
+# Listing your clusters
 
-
-<a name="dcos-dir"></a>
-#### `DCOS_DIR` (DC/OS CLI 0.5.x and higher only)
-
-The path to a DC/OS configuration directory. If you want the DC/OS configuration directory to be `/home/jdoe/config`, set the variable with the command:
-
-```bash
-export DCOS_DIR=/home/jdoe/config
-```
-
-1. Optionally set `DCOS_DIR` and run `dcos cluster setup` command.
-
-    ```
-    export DCOS_DIR=<path/to/config_dir> (optional, default when not set is ~/.dcos)
-    dcos cluster setup <url>
-    ```
-
-   This setting generates and updates per cluster configuration under `$DCOS_DIR/clusters/<cluster_id>`. Sets newly set up cluster as the attached one.
-
-
-<a name="dcos-ssl-verify"></a>
-#### `DCOS_SSL_VERIFY`
-Indicates whether to verify SSL certificates or set the path to the SSL certificates. You must set this variable manually. Setting this environment variable is equivalent to setting the `dcos config set core.ssl_verify` option in the DC/OS configuration [file](#configuration-files). For example, to indicate that you want to set the path to SSL certificates:
+The DC/OS CLI can work with multiple clusters, the one you just configured should now appear with the following command:
 
 ```bash
-export DCOS_SSL_VERIFY=false
+$ dcos cluster list
+        NAME               ID                        STATUS     VERSION      URL
+  *   cluster 26f72c2f-8d03-47d7-b95f-972b1fd3dea2  AVAILABLE    1.12  <cluster-url>
 ```
 
-<a name="dcos-log-level"></a>
-#### `DCOS_LOG_LEVEL`
-Prints log messages to stderr at or above the level indicated. This is equivalent to the `--log-level` command-line option. The severity levels are:
-
-*   **debug** Prints all messages to stderr, including informational, warning, error, and critical.
-*   **info** Prints informational, warning, error, and critical messages to stderr.
-*   **warning** Prints warning, error, and critical messages to stderr.
-*   **error** Prints error and critical messages to stderr.
-*   **critical** Prints only critical messages to stderr.
-
-For example, to set the log level to warning:
+The star next to the cluster name indicates that the CLI is currently attached to it. If you run the setup command again with another cluster, you will see a new item in the list.
```

Comment:
Replace "The star next to the cluster name indicates that the CLI is currently attached to it. "

with 
<sup>*</sup> - Indicates that the CLI is currently attached to the cluster name.